tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/OpenTelemetry/StartRootSpanIntegration.cs

```diff
@@ -0,0 +1,77 @@
+// <copyright file="StartRootSpanIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.OpenTelemetry
+{
+    /// <summary>
+    /// Msmq calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyName = "OpenTelemetry.Api",
+        TypeName = "OpenTelemetry.Trace.Tracer",
+        MethodName = "StartRootSpan",
+        ReturnTypeName = "OpenTelemetry.Trace.TelemetrySpan",
+        ParameterTypeNames = new[] { ClrNames.String, "OpenTelemetry.Trace.SpanKind", "OpenTelemetry.Trace.SpanAttributes", "System.Collections.Generic.IEnumerable`1[OpenTelemetry.Trace.Link]", "System.DateTimeOffset" },
+        MinimumVersion = "1.0.0",
+        MaximumVersion = "1.0.0",
```

Comment:
Why limit ourselves to 1.0.0? Isn't the API already at 1.4?

Reply:
Andrew's correct, the assembly version is at 1.0.0 so this will work for all current versions of the API and I expect it will support all `1.x` NuGet releases that are published